src/applications/claims-status/components/ClaimStatusHeader.jsx

```diff
@@ -1,34 +1,31 @@
-import PropTypes from 'prop-types';
 import React from 'react';
-import { buildDateFormatter, isClaimOpen } from '../utils/helpers';
-
-const isClaimComplete = claim => claim.attributes.status === 'COMPLETE';
+import PropTypes from 'prop-types';
 
-const formatDate = date => buildDateFormatter('MMMM d, yyyy')(date);
+import { DATE_FORMATS } from '../constants';
+import { buildDateFormatter, isClaimOpen } from '../utils/helpers';
 
 const getLastUpdated = claim => {
-  const updatedOn = formatDate(
+  const updatedOn = buildDateFormatter(DATE_FORMATS.LONG_DATE)(
     claim.attributes.claimPhaseDates?.phaseChangeDate,
   );
 
   return `Last updated: ${updatedOn}`;
 };
 
-function ClaimStatusHeader({ claim }) {
+export default function ClaimStatusHeader({ claim }) {
   const { closeDate, status } = claim.attributes;
-  const inProgress = !isClaimComplete(claim) ? 'In Progress' : null;
 
   const isOpen = isClaimOpen(status, closeDate);
 
   return (
     <div className="claim-status-header-container">
       <h2 className="vads-u-margin-y--0">Claim status</h2>
       <p className="vads-u-margin-top--1 vads-u-margin-bottom--3 va-introtext">
-        Here’s the latest information on your claim.{' '}
```

Comment:
I don't think the {' '} is necessary. I inspected the `p` tag in chrome and it is removed from the DOM anyways

Reply:
yea I cant remember why it was added in the first place!